### PR TITLE
introducing AlcaRecos in 2017 WFs

### PR DESCRIPTION
**Pull request by boudoul**
This PR aims to introduce the Alcareco steps (as in run 2 MC) .
It is important to introduce the alcareco in view of MC prodcution, in particular for alignment 

One comment : 
There is one difference with respect to the alca steps for run2 Wfs .
I removed EcalTrg from the alca path which is crashing - Maybe this can be looked later on.

### CODE REVIEW

**Comment by boudoul** [comment]
@mmusich , @franzoni , you may want to take a look to this PR - Thanks

**Comment by mmusich** [comment]
please test

**Comment by mmusich** [comment]
about:

> One comment :
>There is one difference with respect to the alca steps for run2 Wfs .
> I removed EcalTrg from the alca path which is crashing - Maybe this can be looked later on.

needs an update of the phase-I GT, but it's a level of sophistication we don't need at this moment in time.

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -1765,6 +1766,20 @@ def lhegensim(fragment,howMuch):
         upgradeStepDict['HARVESTFull_trackingOnlyPU'][k]=merge([PUDataSets[k2],upgradeStepDict['HARVESTFull_trackingOnly'][k]])
         upgradeStepDict['HARVESTFullGlobalPU'][k]=merge([PUDataSets[k2],upgradeStepDict['HARVESTFullGlobal'][k]])
 
+
+
+    upgradeStepDict['ALCAFull'][k] = {'-s':'ALCA:TkAlMuonIsolated+TkAlMinBias+MuAlOverlaps+EcalESAlign',
```
Comment (mmusich) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: @bsunanda @kpedro88 anything Hcal related you might want to be included in here?

**Comment by mmusich** [comment]
thanks @boudoul !

File: Configuration/PyReleaseValidation/python/relval_steps.py (hunk 1 above)
Comment (kpedro88) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: @bsunanda should comment

**Comment by fabozzi** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_steps.py (hunk 1 above)
Comment (bsunanda) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: It will be nice to add HcalCalHBHEMuonFilter to this

Comment (mmusich) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: @boudoul would you mind adding the request from Sunanda? as I understand this doens't lie anymore in the critical path for pre13..

Comment (boudoul) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: no problem, will update later tonight - thanks for the prompt feedback !

**Comment by davidlange6** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_steps.py (hunk 1 above)
Comment (boudoul) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: This PR has been merged before I could modified it, so the request for the new alcareco is addressed in a new PR #16183

**Pull request merged**